src/ai/backend/manager/models/image.py

```diff
@@ -692,7 +692,7 @@ class Arguments:
 
     ok = graphene.Boolean()
     msg = graphene.String()
-    task_id = graphene.String()
+    tasks = graphene.JSONString()
```

Comment:
I think we should avoid using an arbitrarily structured `JSONString` field here. If it contains just the list of task IDs, it should be something like `graphene.List(graphene.String())`.

Reply:
the task shape is like below. I add this into comment
```json
{
  IMAGE_CANONICAL_1: {
    "bgtask_id": <BACKGROUND_TASK_ID>,
    "image_ref": <IMAGE_CANONICAL>,
    "architecture": <IMAGE_ARCHITECTURE>
  }
  ...
}
```